synapse/cores/common.py

```diff
@@ -568,7 +571,8 @@ def _cortex_condefs():
             ('log:save', {'type': 'bool', 'asloc': 'logsave', 'defval': 0,
                           'doc': 'Enables saving exceptions to the cortex as syn:log nodes'}),
             ('log:level', {'type': 'int', 'asloc': 'loglevel', 'defval': 0, 'doc': 'Filters log events to >= level'}),
-            ('modules', {'defval': (), 'doc': 'An optional list of (pypath,conf) tuples for synapse modules to load'})
+            ('modules', {'defval': (), 'doc': 'An optional list of (pypath,conf) tuples for synapse modules to load'}),
+            ('membranes', {'defval': (), 'doc': 'An optional list of (name,rules) tuples to create Membranes with'})
```

Comment:
add a ``,`` to the end of this conf definition so future additions don't show changes to the previous item in diff view

Reply:
fixed